apps/financial-aid/web-veita/src/components/TableBody/TableBody.tsx

```diff
@@ -53,6 +59,25 @@ const ActivationButtonTableItem = (
   )
 }
 
+const Name = (nationalId: string) => {
```

Comment:
Should this be called pseudoName or something like that or IconAndName, so the user of this component knows what it does without checking out the implementation?